### PR TITLE
Style the bookmark/subscribe button

### PR DESCRIPTION
**Pull request by valenciafu**
image
Looks nicer! Made some changes to SubscribeIcon component based on BookmarkIcon, dunno if I broke anything.

### CODE REVIEW

**Comment by valenciafu** [comment]
Is good to go now! Club page will be slightly restructured and background changed to snow

File: pages/club/[club]/index.js [hunk 1]
```diff
@@ -49,28 +51,42 @@ const Club = ({ query, userInfo, favorites, updateFavorites, subscriptions, upda
...
-          club={club}
-          userInfo={userInfo}
-          favorites={favorites}
-          updateFavorites={updateFavorites}
-          subscriptions={subscriptions}
-          updateSubscriptions={updateSubscriptions}
-          style={{ flex: 1 }}
-        />
-      </Flex>
+    <WideContainer background={`${SNOW}`} fullHeight={true}>
```
Comment (ezwang) [inline pages/club/[club]/index.js]: You should be able to do just `background={SNOW}` instead of ``background={`${SNOW}`}``.

**Comment by cacabo** [comment]
Is this design the most intuitive? not sure it totally communicates what it wants but it's def better than it was

File: components/ClubPage/Actions.js [hunk 2]
```diff
@@ -0,0 +1,124 @@
...
+
+const ActionWrapper = s.span`
+    display: inline-block;
+    flex-direction: row;
+    align-items: center;
+    background-color: ${WHITE};
+    border-radius: 30px;
+    border: 1px solid ${ALLBIRDS_GRAY};
+    padding-left: 1rem;
+    padding-right: 1rem;
```
Comment (cacabo) [inline components/ClubPage/Actions.js]: Indented too much

File: components/ClubPage/Actions.js [hunk 3]
```diff
@@ -0,0 +1,124 @@
...
+`
+
+const ActionDiv = s.span`
+    display: inline-block;
+    color: #4a4a4a;
+    padding-left: 0.5rem;
+    padding-right: 0.5rem;
+    vertical-align: middle;
```
Comment (cacabo) [inline components/ClubPage/Actions.js]: Indented too much

File: components/ClubPage/Actions.js [hunk 4]
```diff
@@ -0,0 +1,124 @@
...
+  const canEdit =
+    (inClub && inClub.role <= ROLE_OFFICER) ||
+    (userInfo && userInfo.is_superuser)
+
+  const [favCount, setFavCount] = useState(club.favorite_count || 0)
+
+  const { code } = club
+
+  return (
+    <div style={style}>
```
Comment (cacabo) [inline components/ClubPage/Actions.js]: would be better to use styled components for this

File: components/common/BookmarkIcon.js [hunk 5]
```diff
@@ -14,6 +13,7 @@ const BookmarkIconTag = s.span`
     float: none;
     position: absolute;
     right: 0;
+
```
Comment (cacabo) [inline components/common/BookmarkIcon.js]:
```suggestion
```

File: pages/club/[club]/index.js (hunk 1 above)
Comment (cacabo) [inline pages/club/[club]/index.js]:
```suggestion
    <WideContainer background={SNOW} fullHeight>
```

File: pages/club/[club]/index.js [hunk 6]
```diff
@@ -49,28 +51,42 @@ const Club = ({ query, userInfo, favorites, updateFavorites, subscriptions, upda
...
+                userInfo={userInfo}
+                favorites={favorites}
+                updateFavorites={updateFavorites}
+                subscriptions={subscriptions}
+                updateSubscriptions={updateSubscriptions}
+                style={{ flex: 1 }}
+              />
+            </Flex>
+          </Card>
+          <Card bordered style={{ marginBottom: '1rem', background: '#ffffff' }}>
```
Comment (cacabo) [inline pages/club/[club]/index.js]:
```suggestion
          <Card bordered style={{ marginBottom: '1rem', background: WHITE }}>
```

File: pages/club/[club]/index.js [hunk 7]
```diff
@@ -49,28 +51,42 @@ const Club = ({ query, userInfo, favorites, updateFavorites, subscriptions, upda
...
+          <Actions
+            club={club}
+            userInfo={userInfo}
+            favorites={favorites}
+            updateFavorites={updateFavorites}
+            subscriptions={subscriptions}
+            updateSubscriptions={updateSubscriptions}
+            style={{ flex: 1 }}
+          />
+          <Card bordered style={{ marginBottom: '1rem', background: '#ffffff' }}>
```
Comment (cacabo) [inline pages/club/[club]/index.js]:
```suggestion
          <Card bordered style={{ marginBottom: '1rem', background: WHITE }}>
```

File: components/ClubPage/Actions.js (hunk 4 above)
Comment (valenciafu) [reply components/ClubPage/Actions.js]: Not sure how, will have to walk thru it later

File: components/common/SubscribeIcon.js [hunk 8]
```diff
@@ -1,31 +1,70 @@
...
+
+  ${({ absolute }) =>
+    absolute &&
+    `
+    float: none;
+    position: absolute;
+    right: 0;
+  `}
+
+    svg {
```
Comment (cacabo) [inline components/common/SubscribeIcon.js]:
```suggestion
  svg {
```

**Comment by cacabo** [comment]
aight is gtg methinks

**Comment by sigmachirality** [comment]
I've pushed changes that allow this PR to also cover the mobile club rework, currently the page looks like this on mobile after the changes:

Screenshot from 2020-01-13 23-41-31

**Comment by sigmachirality** [comment]
@valenciafu is it okay if the buttons are ordered this way instead of the way they were ordered in the mockup, it's a lot easier to code this way.

**Comment by valenciafu** [comment]
@sigmachirality yep, go for it!

**Comment by valenciafu** [comment]
@cacabo pls

**Comment by cacabo** [comment]
going to merge rn, though there is likely room for improvement soon

**Pull request merged**